plugins/modules/lvg_rename.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) Contributors to the Ansible project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+author:
+  - Laszlo Szomor (@lszomor)
+module: lvg_rename
+short_description: Renames LVM volume groups
+description:
+  - This module renames volume groups using the C(vgchange) command.
+extends_documentation_fragment:
+  - community.general.attributes
+attributes:
+  check_mode:
+    support: full
+  diff_mode:
+    support: full
+version_added: 7.1.0
+options:
+  vg:
+    description:
+    - The name or UUID of the source VG.
+    - See V(vgrename(8\)) for valid values.
+    type: str
+    required: true
+    aliases:
+      - name
+      - vg_uuid
```

Comment:
We should generally avoid adding aliases except in situations where it is really necessary. I would remove both aliases. If there's an alias, I would use `vg_old` (or even rename the module option to that and use no alias).

Reply:
Thank you, I'll remove the aliases. 
I would stay with the `vg` option naming, this is consistent with the `lvg` and `lvol` modules.